cmd/pw_router/worker.go

```diff
@@ -50,7 +52,7 @@ func (w *worker) isSignificant(last export.PlaneLocation, candidate export.Plane
 		return true
 	}
 
-	if candidate.HasVerticalRate && last.HasVerticalRate && candidate.VerticalRate != last.VerticalRate {
+	if candidate.HasVerticalRate && last.HasVerticalRate && math.Abs(float64(candidate.VerticalRate-last.VerticalRate)) > SigVerticalRateChange {
```

Comment:
Do we want to have this only above a certain altitude? 180ft could be a significant enough change below 2000ft for example?

Reply:
@bluntelk this is VerticalRate, as the altitude is changing closer to the ground, that will cause us to emit a significant frame. This is more to mute out "wobble" in fpm while a plane is in cruise. E.g. if a plane dips 2ft in the air during cruise, we get a frame with -120fpm. The 10ft of "SigAltitudeChange" will work with this too.